packages/go_router/lib/src/configuration.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'configuration.dart';
+import 'logging.dart';
+import 'path_utils.dart';
+import 'typedefs.dart';
+
+export 'route.dart';
+export 'state.dart';
+
+/// The route configuration for GoRouter, configured by the app
+class RouteConfiguration {
+  /// Constructs a RouterConfiguration
+  RouteConfiguration({
+    required this.routes,
+    required this.redirectLimit,
+    required this.topRedirect,
+  }) {
+    _cacheNameToPath('', routes);
+    assert(() {
+      _debugKnownRoutes();
+      return true;
+    }());
+  }
+
+  /// List of top level routes used by the go router delegate
```

Comment:
It looks like periods were removed from the end of all comments when this was moved out of `GoRouteInformationParser`. Is that intentional? My understanding is the Flutter style guide is to always have periods at the end of comments.

Reply:
SGTM - I went ahead and added periods to all of the doc comments.